modules/rosa-sts-ocm-roles-and-permissions.adoc

```diff
@@ -0,0 +1,102 @@
+//
+// rosa_architecture/rosa-sts-understanding-roles-and-permissions.adoc
+//
+:_content-type: PROCEDURE
+[id="rosa-sts-ocm-roles-and-permissions_{context}"]
+= {cluster-manager} roles and permissions
+
+When creating clusters, you must have at least three roles to create and use the clusters. These roles are:
+
+* A user role
+* A basic `ocm-role` resource
+* An admin `ocm-role` resource
+
+[id="rosa-sts-ocm-roles-and-permissions-iam-user-role_{context}"]
+== User role
+
+The user role has no additional permissions. {cluster-manager} identifies this user role and creates trust between this role, the Red Hat account, and the two `ocm-role` resources that you create.
+
+//Used to identify the user by creating a trust between it, Red Hat account and the ocm role.
+
+[id="rosa-sts-ocm-roles-and-permissions-iam-basic-role_{context}"]
+== Creating basic {cluster-manager} permissions
+
+The basic {cluster-manager} role permissions allow you to perform cluster maintenance within {cluster-manager}.
+
+.Procedure
+* Create a basic `ocm-role` resource:
++
+[source,terminal]
+----
+rosa create ocm-role
+----
++
+This command creates the role, prompting you for specifications for the role. The basic `ocm-role` resource contains the following permissions:
++
+.Associated permissions for the basic `ocm-role`
+[cols="1,2",options="header"]
+|===
+
+|Resource|Description
+
+| `iam:GetOpenIDConnectProvider`
+| This permission allows the basic role to retrieve information about the specified OpenID Connect (OIDC) provider.
+| `iam:GetRole`
+| This permission allows the basic role to retrieve any information for a specified role. Some of the data returned include the role's path, GUID, ARN, and the role's trust policy that grants permission to assume the role.
+| `iam:ListRoles`
+| This permission allows the basic role to list the roles within a path prefix.
+| `iam:ListRoleTags`
+| This permission allows the basic role to list the tags on a specified role.
+| `ec2:DescribeRegions`
+| This permission allows the basic role to return information about all of the enabled regions on your account.
+| `ec2:DescribeRouteTables`
+| This permission allows the basic role to return information about all of your route tables.
+| `ec2:DescribeSubnets`
+| This permission allows the basic role to return information about all of your subnets.
+| `ec2:DescribeVpcs`
+| This permission allows the basic role to return information about all of your virtual private clouds (VPCs).
+| `sts:AssumeRole`
+| This permission allows the basic role to retrieve temporary security credentials so that you you can use to access AWS resources that are beyond your normal permissions.
+| `sts:AssumeRoleWithWebIdentity`
+| This permission allows the basic role to retrieve temporary security credentials for users authenticated their account with a web identity provider.
+
+|===
+
+[id="rosa-sts-ocm-roles-and-permissions-iam-admin-role__{context}"]
+== Creating admin {cluster-manager} permissions
```

Comment:
You could consider making each of this subsections a separate AsciiDoc module and changing the offset level increments in the include statements for each. This will give us more granular reuse of the modules in future PRs.